.circleci/test-deploy.yml

```diff
@@ -0,0 +1,147 @@
+---
+version: 2.1
+
+# ----------------
+# Orbs declaration
+# ----------------
+
+orbs:
+  noos-ci: noosenergy/noos-ci@dev:<<pipeline.git.revision>>
```

Comment:
Acoording to the doc templates (and as per the official Slack Orb), such a parameter should be left empty - injected at execution time.

https://github.com/CircleCI-Public/slack-orb/blob/main/.circleci/test-deploy.yml#L4

https://github.com/CircleCI-Public/Orb-Template/blob/main/.circleci/test-deploy.yml#L7

Reply:
When I leave it empty, I get this error

<img width="970" alt="image" src="https://github.com/noosenergy/noos-circleci-orb/assets/12544657/765e250c-7510-41a8-b202-f56601ede2ef">

Reply:
https://circleci.com/docs/testing-orbs/

https://github.com/KyleTryon/calculator-orb